auslib/web/admin/views/scheduled_changes.py

```diff
@@ -123,7 +121,7 @@ def _delete(self, sc_id, transaction, changed_by):
         if not sc:
             return Response(status=404, response="Scheduled change does not exist")
 
-        form = DbEditableForm(request.args)
+        form = DbEditableForm(connexion.request.args)
```

Comment:
this particular form is being used for "/scheduled_changes/:namespace/:sc_id" endpoints. These endpoints will be migrated in the upcoming PR. Then once all the form-usage instances have been purged from the code, we can safely delete the models from forms.py and by the end of all API migrations, the entire forms.py file